### ISSUE TITLE
Wrong value search for editableTypeSelect

### ISSUE THREAD

**Issue by holantomas** [ublaboo/datagrid#596]
I think, [this](https://github.com/ublaboo/datagrid/blob/v5.x/assets/src/datagrid.coffee#L511) is wrong cause you are searching for `<option>` with Text=Value and then search again `<option>` with value.

JS side just excepting that option's text=value but PHP side have ability to setup different value from text.

[Here](https://github.com/ublaboo/datagrid/blob/v5.x/src/Column/Column.php#L624) 
```php
		foreach ($options as $value => $text) {
			$select->create('option')
				->value($value)
				->setText($text);
		}
```

And for second this, cause error which is mentioned in this [issue](https://github.com/ublaboo/datagrid/issues/572)

**Comment by holantomas** [comment]
Any idea?

**Comment by paveljanda** [comment]
This is a duplicit issue and will be solved in following days via #573
